cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -135,13 +135,20 @@ func WriteKubeletConfigFiles(cfg *kubeadmapi.InitConfiguration, kubeletConfigDir
 		_, err := os.Stat(filepath.Join(kubeletDir, kubeadmconstants.KubeletInstanceConfigurationFileName))
 		if os.IsNotExist(err) {
 			var containerRuntimeEndpoint string
+			var kubeletFlags []kubeadmapi.Arg
 			dynamicFlags, err := kubeletphase.ReadKubeletDynamicEnvFile(filepath.Join(kubeletDir, kubeadmconstants.KubeletEnvFileName))
 			if err == nil {
 				args := kubeadmutil.ArgumentsFromCommand(dynamicFlags)
 				for _, arg := range args {
 					if arg.Name == "container-runtime-endpoint" {
 						containerRuntimeEndpoint = arg.Value
-						break
+					} else {
+						kubeletFlags = append(kubeletFlags, arg)
+					}
```

Comment:
```
        containerRuntimeEndpoint = arg.Value
        continue
    }
    kubeletFlags = append(kubeletFlags, arg)
}
```

Reply:
I added `continue`, will this make the program run faster, or is it purely for readability optimization? The code in the `else` block wouldn’t run even without the `continue`, but I haven’t looked into whether the compiler optimizes for this scenario.